src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -299,6 +309,21 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
   {
     model->Lambda() = lambda;
 
+    // Did we want training accuracy?
+    if (params.Has("print_training_accuracy"))
+    {
+      timers.Start("lr_prediction");
+      arma::Row<size_t> predictions;
+      model->Classify(regressors, predictions);
+
+      const size_t correct = arma::accu(predictions == responses);
+
+      Log::Info << correct << " of " << responses.n_elem << " correct on training"
+          << " set (" << (double(correct) / double(responses.n_elem) * 100) << ")."
+          << endl;
+      timers.Stop("lr_prediction");
+    }
```

Comment:
Thanks for moving the block, but we can't print accuracy before we train the model.  Please move it down below the training code (to the end of this `if` block).